test/unit/APACitation.test.js

```diff
@@ -0,0 +1,234 @@
+
+    it('should turn names into a comma-delimited list of initialized names', () => {
+      const agentNames = instance.formatAgentNames('testing');
+      expect(agentNames).to.equal('Test, A., Test, O., Test, S. N., Test, S. A.');
+    });
+
+    it('should turn names into a comma-delimited list of initialized names, with initial first', () => {
+      const agentNames = instance.formatAgentNames('testing', true);
+      expect(agentNames).to.equal('A. Test, O. Test, S. N. Test, S. A. Test');
+    });
+  });
+
+  describe('returnMonographCitation()', () => {
+    let component;
+    it('should return formatted monograph citation', () => {
+      component = mount(<APACitation
+        title="Testing"
+        subTitle="subTest"
+        agents={{
+          authors: ['Author, Test'],
+          editors: ['Editor, Test'],
+          translators: ['Translator, Test'],
+          illustrators: ['Illustrator, Test'],
+          publishers: ['Test Publisher'],
+        }}
+        publicationYear="2020"
+        edition="Test Edition"
+        volume={null}
+        sourceLink="test_link"
+        isGovernmentDoc={false}
+      />);
+      const citationText = component.find('.apa-citation').text();
+      // eslint-disable-next-line max-len
+      const outText = 'APAAuthor, T. (2020). Testing: subTest (T. Illustrator, Illus., T. Translator, Trans.) (Test Edition) (T. Editor, (Ed.)). Test Publisher. test_link';
+      expect(citationText).to.equal(outText);
+    });
+
+    it('should return formatted monograph citation with editors if no authors', () => {
+      component = mount(<APACitation
+        title="Testing"
+        subTitle="subTest"
+        agents={{
+          authors: [],
+          editors: ['Editor, Test'],
+          translators: ['Translator, Test'],
+          illustrators: [],
+          publishers: ['Test Publisher'],
+        }}
+        publicationYear="2020"
+        edition="Test Edition"
+        volume={null}
+        sourceLink="test_link"
+        isGovernmentDoc={false}
+      />);
+      const citationText = component.find('.apa-citation').text();
+      // eslint-disable-next-line max-len
+      const outText = 'APAEditor, T., (Ed.) (2020). Testing: subTest (T. Translator, Trans.) (Test Edition) Test Publisher. test_link';
+      expect(citationText).to.equal(outText);
+    });
+
+    it('should return formatted monograph citation with volume if included', () => {
+      component = mount(<APACitation
+        title="Testing"
+        subTitle="subTest"
+        agents={{
+          authors: ['Author, Test'],
+          editors: [],
+          translators: [],
+          illustrators: [],
+          publishers: ['Test Publisher'],
+        }}
+        publicationYear="2020"
+        edition={null}
+        volume="Vol. 2"
+        sourceLink="test_link"
+        isGovernmentDoc={false}
+      />);
+      const citationText = component.find('.apa-citation').text();
+      // eslint-disable-next-line max-len
```

Comment:
Why does this need to be disabled?  (If it's too long you can concatenate strings across 2 lines)

Reply:
It doesn't I just added it while I was working, copied the code and then forgot to take it out.